vsphere/datadog_checks/vsphere/cache.py

```diff
@@ -148,3 +148,10 @@ def set_mor_props(self, mor, mor_data):
         if mor_type not in self._mors:
             self._mors[mor_type] = {}
         self._mors[mor_type][mor] = mor_data
+
+    def clear_properties(self):
+        # type: () -> None
+        for _, mors in self._mors.items():
+            for _, mor_props in mors.items():
+                if mor_props.get('properties') is not None:
+                    del mor_props['properties']
```

Comment:
nit - This is equivalent to `mor_props.pop('properties', None)` (similar to what you did previously)

Reply:
Updated!